runner/run/execute.go

```diff
@@ -184,22 +186,33 @@ func excludeLine(excludes []string, line string) bool {
 	return false
 }
 
-func buildIssueComment(cfg *Config, outputs []string) string {
-	var s strings.Builder
-	s.WriteString(fmt.Sprintf("Run `%s` on Repo: %s\n", cfg.LinterCfg.LinterCommand, cfg.Repo))
-	s.WriteString("\n")
-	s.WriteString(fmt.Sprintf("Got total %d line output in action: %s", len(outputs), os.Getenv("GH_ACTION_LINK")))
-	s.WriteString("\n")
-	s.WriteString("<details>\n")
-	s.WriteString("<summary>Expand</summary>\n\n")
-	for i, line := range outputs {
+//go:embed templates/issue_comment.md
+var issueCommentTemplate string
+
+type issueCommentData struct{
+	GithubActionLink string
+	Lines            []string
+	Linter           string
+	RepositoryURL    string
+}
+
+func buildIssueComment(cfg *Config, outputs []string) (string, error) {
+	var data = &issueCommentData{
+		GithubActionLink: os.Getenv("GH_ACTION_LINK"),
+		Linter: cfg.LinterCfg.LinterCommand,
+		RepositoryURL: cfg.Repo,
+		
+	}
+	for _, line := range outputs {
 		text := buildIssueCommentLine(cfg, line)
-		s.WriteString(fmt.Sprintf("%d. %s\n", i+1, text))
+		data.Lines = append(data.Lines, text)
+	}
+	var tpl bytes.Buffer
+	tmpl := template.Must(template.New("issue_comment").Parse(issueCommentTemplate))
```

Comment:
I'm unsure about the Must here

I know it's open to debate.

```suggestion
	tmpl, err := template.New("issue_comment").Parse(issueCommentTemplate)

  if err != nil {
       return "", err
  }
```

Reply:
Thanks for applying my suggestion, but I'm afraid the code I suggested via my phone needs to be gofmt-ed